chsdi/templates/luftbilder/viewer.mako

```diff
@@ -260,7 +282,47 @@
         view.on('propertychange', debounce(updateUrl));
         updateUrl();
 
+        var contrastSlider = document.getElementById("contrast");
+        var contrastOut = document.getElementById("contrastOut");
+        var resetbutton = document.getElementById("reset");
+
+        setSliderListeners();
+
+        raster.on('beforeoperations', function(event) {
+          var data = event.data;
+          data["contrast"] = Number(contrastSlider.value);
+        });
+
+        function onSliderChange() {
+          contrastOut.innerHTML = contrastSlider.value + "%";
+          raster.changed();
+        }
+
+        function onReset(){
+          var contrastSlider = document.getElementById("contrast");
+          contrastSlider.value = 100;
+          var percent = document.getElementById("contrastOut");
+          percent.innerHTML = "100%";
```

Comment:
Isn't  the 'change' event triggered when you set the value to 100?

Reply:
I also expected that but it didn't and I don't understand why. That's why I added the `raster.changed()`.